CodeStarterCamp_Week2/File.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  File.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by 이선준 on 2023/03/10.
+//
+
+//MARK: STEP2
+
+import Foundation
+
+func createWinningLottoNumbers() -> Set<Int> {
+    var lottoNumberSet: Set<Int> = Set<Int>()
+    while lottoNumberSet.count < 6 {
+        lottoNumberSet.insert(Int.random(in: 1...45))
+    }
+    print(lottoNumberSet)
+    return lottoNumberSet
+}
+
+func compareLottoNumber(_ winningLottoNumbers: Set<Int>, _ myLottoNumbers: [Int]) {
+    let intersectionNumbers: Array<Int> = winningLottoNumbers.intersection(myLottoNumbers).sorted()
```

Comment:
요건 사바사 입니다만, 저는 배열을 선언할 때 요런식으로 표현하는 편입니다. 😊
```suggestion
    let intersectionNumbers: [Int] = winningLottoNumbers.intersection(myLottoNumbers).sorted()
```